aspnetcore/mvc/views/tag-helpers/builtin.md

```diff
@@ -11,7 +11,7 @@ Microsoft has included 17 Tag Helpers that can be immediately used in .net Core
 
 **[AnchorTagHelper](builtin/resources/AnchorTagHelper.md)**
 
-[comment]: **[CacheTagHelper](builtin/resources/AnchorTagHelper.md)**
+**[CacheTagHelper](builtin/resources/CacheTagHelper.md)**
```

Comment:
@Rick-Anderson suggested by following the format of the doc listed in  https://github.com/aspnet/Docs/pull/2644#issuecomment-277123347 that the TH's would be plainly named for reading purposes; but of course, they'll say what they want to do there. I think they read better in plain language.